README.md

```diff
@@ -628,6 +628,7 @@ Table of Contents
 ## Web Hosting
 
   * [000WebHost](https://www.000webhost.com/) — Zero cost website hosting with PHP, MySQL, cPanel & no ads!
+  * [Amezmo](https://www.amezmo.com) — PHP hosting for developers. Docker container based. Modern PHP 7, Automatic Deployments with Git, free SSL for custom domains. 
```

Comment:
whats the details of the free tier on this? I don't see any free tier.

Reply:
I found. https://www.amezmo.com/pricing#accordion-1

![](https://user-images.githubusercontent.com/60031799/81536123-90914280-935a-11ea-828f-0ebf0e6cc7b7.png)

Reply:
OK, but its not really clear what that means - what features are available etc? Looking at other entries here you'll see there's limits and so forth mentioned.

As it is, sure there's a free tier, but I they need to be a bit more clear about what that means

Reply:
Hi @ripienaar Please check out https://www.amezmo.com/pricing for the updated Free Tier plan description.

Reply:
excellent, please rebase and we can merge